debian/control

```diff
@@ -40,6 +40,7 @@ Standards-Version: 4.1.2
 
 Package: nautilus
 Architecture: any
+X-Eos-Appid: org.gnome.Nautilus
```

Comment:
Can you make this `XBS-`? It doesn't matter too much, but that way it shows up in our package index and then it's easier to see where we have these kinds of overrides.